net/dns/resolved.go

```diff
@@ -167,11 +168,30 @@ func (m *resolvedManager) SetDNS(config OSConfig) error {
 		})
 	}
 
-	err = m.resolved.CallWithContext(
-		ctx, "org.freedesktop.resolve1.Manager.SetLinkDomains", 0,
-		m.ifidx, linkDomains,
-	).Store()
-	if err != nil {
+	for try := 1; try < 2; try++ {
+		if try == 2 {
```

Comment:
I'd prefer to match on the specific error and only retry in that case, otherwise we might end up with errors like "oops not ready yet" leading us to blindly setting a degraded config.

Reply:
done